apps/i18n/common/en_us.json

```diff
@@ -365,6 +367,7 @@
   "copyright": "Copyright",
   "correct": "Correct",
   "correctAnswer": "That is the correct answer.",
+  "costume": "costume",
```

Comment:
just a heads up to @epeach -- i think you'll be adding some similar strings for another animation library dialog, so just an FYI that you'll likely be able to use the ones mike is adding here